common/utils/consoleAdvertisement.ts

```diff
@@ -2,26 +2,31 @@
 export default function consoleAdvertisement() {
   console.log(
     `%c
-                                                                                                     
-                    ,:;;;;;;;;;;:.                                                                   
-                .;;;;;;::;;;;;;;;;;;;'                                                               
-              ::::;;;;::::;;;;;;;;;;;;;:               _       _         _    _     _                
-            ;::;;::;;:::'    .;;;;;;;;;;;,            | |     (_)       | |  | |   | |               
-           ;;:;;;;::;:;           ;;;;;;;;:           | | ___  _ _ __   | |  | |___| |               
-          ;;:;;;;;;::;;;            ;;;;;;;:      _   | |/ _ \\| | '_ \\  | |  | / __| |               
-         ;;::;;;;;::::;;;;;;;,       ;;;;;;;,    | |__| | (_) | | | | | | |__| \\__ |_|               
-         ;::;;;;  :;;::;;;;;::;;      ;;;;;;;     \\____/ \\___/|_|_| |_|  \\____/|___(_)               
-        :::;;;;'     ;:;;;;;;::;'     ,;;;;;;                                                        
-        :;;;;;;      ;::;;;;'         ,;;;;::    https://github.com/MyEtherWallet/MyEtherWallet      
-        .;;;;;;,     ,;::;;;;;;;;     ;;;;;:;                                                        
-         ;;;;;;;       ;::;;;;;;:;:;',;;;;:;;                                                        
-         ';;;;;;;           ';;;;::;;;;;;::;                                                         
-          .;;;;;;;'           ;;:::;;;;;::;                                                          
-            ;;;;;;;;;''        ;:;;:;;;::;                                                           
-              ::::;;;;::::;;;;;;;;;;;;;:                                                             
-                .;;;;;;::;;;;;;;;;;;;'                                                               
-                    ,:;;;;;;;;;;:.                                                                   
-                                                                                                     
+               NNmdhhyyyyyyyyhhddmN               
+            mdhyyyyyyyyyyyyyyyyyyyyyhdN           
+         mdyyyyyyyyyyyyyyyyyyyyyyyyyyyyhdN        
+      Ndhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhm      
+    Nmhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhm       
+    dyyyyyyyyyyys++oyyyyyyyyyys+/+syyyyyyyyyyhd   
+  Nhyyyyyyyyyy+.    .+yyyyyyo-     :syyyyyyyyyhdN          _       _         _    _     _ 
+ Nhyyyyyyyyyo.        .+yyo-         :yyyyyyyyyyd         | |     (_)       | |  | |   | |
+ hyyyyyyyyo.            .-             :yyyyyyyyhm        | | ___  _ _ __   | |  | |___| |
+myyyyyyyy.     -+oo/.        .+ooo:`    `oyyyyyyyh    _   | |/ _ \| | '_ \  | |  | / __| |
```

Comment:
I haven't run this yet so I might be wrong but it looks like there's an a backtick in here which is escaping the string. Does this work?